src/network/rpc/EthClient.h

```diff
@@ -0,0 +1,384 @@
+/**
+ * This file is generated by jsonrpcstub, DO NOT CHANGE IT MANUALLY!
+ */
+
+#ifndef JSONRPC_CPP_STUB_TARAXA_NET_ETHCLIENT_H_
+#define JSONRPC_CPP_STUB_TARAXA_NET_ETHCLIENT_H_
+
+#include <jsonrpccpp/client.h>
+
+namespace taraxa {
+    namespace net {
```

Comment:
we are using 2 space padding in whole codebase

Reply:
this is a generated code

Reply:
so there is no possibility to set 2 space padding in tool that generated it ?

Reply:
done - included `.h` files for clang-format